### PR TITLE
Fix import/export bugs and add a few tests

### LINKED ISSUE #211: [PE-D] Bugs on `show` command
The file is successfully saved with

but fails with

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: eltonyeh/ped#7

### LINKED ISSUE #218: [PE-D] Bugs on `import`
The message returned by the app does not match the actual error. It should return something about the wrong file extension.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: eltonyeh/ped#10

### LINKED ISSUE #219: [PE-D] Bug about `export`
Exporting to files with invalid filenames results in no response from the software

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: eltonyeh/ped#11

### LINKED ISSUE #226: [PE-D] Show command does not execute with no error message when invalid file path is provided
When using the show command with an invalid specified file path, the command is not executed when pressing enter, and no error message is shown. 

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: jyrw/ped#10

### LINKED ISSUE #229: [PE-D] Error message not stated correctly for import command
### Suggested Fix:

`Failed to read number of columns` would be a more suitable error message since `-g` and `-a` here are not for specifying the group or assessment but rather the number of columns 

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: Jacobnbh/ped#4

### LINKED ISSUE #231: [PE-D] Exporting a .csv file
Exporting to a file called `.csv` will make the file not work with Excel, but it works with any text editor.

Maybe a warning can be shown to let the user be aware of this.

Command used

```
export -f .csv
```

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: Gabau/ped#3

### PR DESCRIPTION
**Pull request by jonas-chow**
Closes #211, closes #218, closes #219, closes #226, closes #229, closes #231

Removed the ability to specify file paths for export and show command.
Also added more checks to the specified file path for the import command.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -28,6 +29,9 @@
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
     public static final String MESSAGE_EMPTY_PATH = "Path should not be empty";
+    public static final String MESSAGE_INVALID_PATH = "Path provided is invalid";
+    public static final String MESSAGE_DIRECTORY = "Path provided should not be for a directory";
```
Comment (zhing22) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]:
```suggestion
    public static final String MESSAGE_EMPTY_PATH = "Path should not be empty. ";
    public static final String MESSAGE_INVALID_PATH = "Path provided is invalid. ";
    public static final String MESSAGE_DIRECTORY = "Path provided should not be for a directory. ";
```
thanks!

File: src/main/java/seedu/address/logic/parser/ShowCommandParser.java [hunk 2]
```diff
@@ -30,7 +31,7 @@ public ShowCommand parse(String args) throws ParseException {
 
         Path savePath = null;
         if (argMultimap.getValue(PREFIX_FILE).isPresent()) {
-            savePath = ParserUtil.parsePath(argMultimap.getValue(PREFIX_FILE).get(), ".png");
+            savePath = generateNewPath(0);
```
Comment (ganhongyao) [inline src/main/java/seedu/address/logic/parser/ShowCommandParser.java:34]: Does this mean that we check that the `-f` flag was specified in the command, and if it is, we export the graph with our own file name? And whatever argument comes after `-f` gets ignored?

**Review by ganhongyao — COMMENTED** [review]
Also, should we update the various `MESSAGE_USAGE`s and the user guide in this PR?

Otherwise LGTM, tested and no longer face the same issues.

File: src/main/java/seedu/address/logic/parser/ShowCommandParser.java (hunk 2 above)
Comment (jonas-chow) [reply src/main/java/seedu/address/logic/parser/ShowCommandParser.java:34]: Yes, that is exactly what this means. Any input between `-f` and the next prefix is ignored. (as weird as that may be)

Not sure if there's any other way we can allow the user to specify they want to save the graph without allowing them to input the path themselves. Adding a `exportgraph` command maybe? But that's treading the line of "no new features".

**Comment by ganhongyao** [comment]
LGTM

**Review by ganhongyao — APPROVED** [review]

**Pull request merged**